microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -170,14 +170,16 @@ Each image defines one service.  Some images don't contain UDFs, and consist ent
 
 Microsoft provides an image for each service, which can be deployed on a cloud provider.  These are the public images that must be used for deployment.  Only official images from Microsoft are able to run private auctions.
 
-Images for sellers:
-* [Ad Selection Seller Front End Service](https://mcr.microsoft.com/product/ad-selection/azure/seller-frontend-service)
-* [Ad Selection Auction Service](https://mcr.microsoft.com/product/ad-selection/azure/auction-service)
-
-Images for buyers:
-* [Ad Selection Buyer Front End Service](https://mcr.microsoft.com/product/ad-selection/azure/buyer-frontend-service)
-* [Ad Selection Bidding Service](https://mcr.microsoft.com/product/ad-selection/azure/bidding-service)
-* [Ad Selection K-Anonymity Service](https://mcr.microsoft.com/product/ad-selection/azure/k-anonymity-service)
+Images for deployment:
+* Sellers
+   * [Ad Selection Seller Front End Service](https://mcr.microsoft.com/product/ad-selection/azure/seller-frontend-service)
```

Comment:
maybe for future PR:
in service-image page titles and in present article globally, make name of services consistent re: space chars, cap'zn, & bold where applicable.  eg everywhere, say either:
* the **Seller Front End** service
* the **SellerFrontEnd** service
@yanushP @captainbrosset

Reply:
also check usage in articles (how the name of service is presented/formatted) about Ad Selection API in other repo.
@yanushP @captainbrosset